index.html

```diff
@@ -0,0 +1,44 @@
+<!DOCTYPE html>
+<html lang="en">
+<head>
+    <meta charset="UTF-8">
+    <meta http-equiv="X-UA-Compatible" content="IE=edge">
+    <meta name="viewport" content="width=device-width, initial-scale=1.0">
+    <title>Document</title>
+    <link rel="stylesheet" href="style.css">
+</head>
+<body>
+    <div class="container">
+        <div class="box header">
+            <span>My Logo</span>
+            <span><img src="img/Enabled.png"></span>
+        </div>
+        <div class="box content">
```

Comment:
- [ ] I think it would be better to use semantic HTML5 tags, so I suggest using <section class="box content"> tag instead of <div class="box content"> 👌